nerdtree_plugin/fs_menu.vim

```diff
@@ -225,10 +225,12 @@ endfunction
 function! NERDTreeListNode()
     let treenode = g:NERDTreeFileNode.GetSelected()
     if !empty(treenode)
-        if has("osx")
-            let stat_cmd = 'stat -f "%z" '
-        else
-            let stat_cmd = 'stat -c "%s" '
+        if has("unix")
```

Comment:
Taking a wider look at this file, I noticed that this `if has("unix")` statement has already been done on line 41, so we don't need it again here.